package.json

```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@chakra-ui/react": "2.5.4",
-    "@newrelic/next": "^0.7.0",
+    "@newrelic/next": "^0.6.0",
```

Comment:
The version for these packages shouldn't need to be changed for this PR.

Reply:
@jackiequach undid the changes. 

Thanks